.travis.yml

```
@@ -88,11 +88,11 @@ deploy:
     skip_cleanup: true
     script: cd $TRAVIS_BUILD_DIR; ./build-docker-inspirehep-image.sh
     on:
-      branch: master
-      condition: $SUITE = integration
+      all_branches: true
+      condition: $SUITE = integration AND $TRAVIS_BRANCH =~ ^master|prod$
```

are you sure this shouldn't be
```suggestion
      condition: $SUITE = integration && $TRAVIS_BRANCH =~ ^master|prod$
```
https://docs.travis-ci.com/user/deployment says it's evaluated through `if [[ <condition> ]]` in the shell, but those use `&&`. That's how we do it in hepcrawl: https://github.com/inspirehep/hepcrawl/blob/5b1e0ab16c9bace40450544fa5d5a5fb6fc17ff9/.travis.yml#L77

---

as far as I understand from the docs it should work https://docs.travis-ci.com/user/conditions-v1#boolean-operators but I just change it to ``&&``